istioctl/cmd/add-to-mesh.go

```diff
@@ -230,6 +228,19 @@ See also 'istioctl experimental remove-from-mesh service' which does the reverse
 	return cmd
 }
 
+func injectSideCarIntoDeployments(client kubernetes.Interface, deps []appsv1.Deployment, sidecarTemplate, valuesConfig,
```

Comment:
This might be a better naming, but no optimization for the code path if I understand correctly.

Reply:
Basically yes